internal/pkg/manifest/errors.go

```diff
@@ -50,3 +53,40 @@ func (e *ErrUnknownProvider) Is(target error) bool {
 	_, ok := target.(*ErrUnknownProvider)
 	return ok
 }
+
+type errFieldMustBeSpecified struct {
+	missingField      string
+	conditionalFields []string
+}
+
+func (e *errFieldMustBeSpecified) Error() string {
+	errMsg := fmt.Sprintf(`"%s" must be specified`, e.missingField)
+	if len(e.conditionalFields) == 0 {
+		return errMsg
+	}
+	return fmt.Sprintf(`%s if "%s" %s specified`, errMsg, strings.Join(e.conditionalFields, `", "`),
+		english.PluralWord(len(e.conditionalFields), "is", "are"))
+}
+
+type errFieldMutualExclusive struct {
+	firstField  string
+	secondField string
+	mustExist   bool
+}
+
+func (e *errFieldMutualExclusive) Error() string {
+	var s string
+	if e.mustExist {
+		s = " not none,"
+	}
+	return fmt.Sprintf(`must specify one, not both,%s of "%s" and "%s"`, s, e.firstField, e.secondField)
```

Comment:
My pea brain is having difficulty parsing this sentence with `mustExist` 😂 

What do you think of creating a new error type instead
```go
type errFieldsMissing struct {
  requiredFields []string
}
func (e *errFieldsMissing) Error() string { return "must specify one of ..." }
```

Reply:
`errFieldsMissing` it seems to be missing all the fields in it 🤔  How do we differentiate it from "must specify all of"? Maybe i'll make the code more readable instead!